project/src/components/offer-card/offer-card.tsx

```diff
@@ -1,61 +1,51 @@
 import { Link, generatePath } from 'react-router-dom';
 import cn from 'classnames';
 
+import Bookmark from '../bookmark/bookmark';
+import Badge from '../badge/badge';
+
 import { AppRoute } from '../../const';
 
 import { Offer } from '../../types/offer';
 
 type PropsType = {
   offer: Offer;
+  className: string;
   onOfferCardMouseEnter: (id: number) => void;
-  activeCard: number;
-  page: string;
+  isSmall?: boolean;
 }
 
-function OfferCard({offer, page, onOfferCardMouseEnter, activeCard}: PropsType): JSX.Element {
+function OfferCard({offer, className, onOfferCardMouseEnter, isSmall}: PropsType): JSX.Element {
   const {id, price, isPremium, isFavorite, imgPath, title, roomType} = offer;
-  const favoriteClassName = 'place-card__bookmark-button button';
 
-  const handleMouseEnter = () => {
-    onOfferCardMouseEnter(id);
-  };
+  const imgWidth = !isSmall ? '260' : '150';
```

Comment:
Я обычно стараюсь писать неинвентированные условия. Менять или нет сам выбери
```suggestion
  const imgWidth = isSmall ? '150' : '260';
```